### ISSUE TITLE
metric to guide Trainer1D sequence denoising

### ISSUE THREAD

**Issue by johnnytam100** [lucidrains/denoising-diffusion-pytorch#341]
Hi @lucidrains ! Great implementation!
I want to use Trainer1D to guide sequence denoising.
Would you mind giving a very brief example of incorporating a custom metric into the loss function?
Thanks in advance!
